src/features/identityCheck/pages/helpers/useSubscriptionNavigation.ts

```diff
@@ -34,13 +38,14 @@ export const useSubscriptionNavigation = (): {
 
   const { mutateAsync: patchProfile } = usePatchProfile()
 
-  const saveCheckpoint = async (nextStep: IdentityCheckStep) => {
+  const saveCheckpoint = async (nextStep: IdentityCheckStep | IdentityCheckStepNewStepper) => {
     try {
-      if (currentStep === IdentityCheckStep.PROFILE) {
+      if (currentStep === (IdentityCheckStep.PROFILE || IdentityCheckStepNewStepper.PROFILE)) {
         setIsSavingCheckpoint(true)
         await patchProfile()
       }
       await queryClient.invalidateQueries(QueryKeys.NEXT_SUBSCRIPTION_STEP)
+      await queryClient.invalidateQueries(QueryKeys.STEPPER_INFO)
```

Comment:
Finalement ce commit invalide la requête seulement à la fin de l'étape du profil et non "at the end of each step" ?

Reply:
Ah zut bien vu ! Je l'avais fait que là pour tester que ça marchait mais ça ne suffit clairement pas !